src/components/forms/verify-two-fa-code-form.tsx

```diff
@@ -57,7 +57,11 @@ const VerifyTwoFACodeForm: React.FC<{
           }
 
           if ("isCodeVerified" in parsedResponse) {
-            router && router.push("/account/security/update");
+            if (router.query.next && typeof router.query.next === "string") {
+              router.push(router.query.next);
+            } else {
+              router.push("/");
```

Comment:
Actually, could you change this to `/account` ?

Reply:
Done 👍